tests/external/test_stereoscope.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+from scvi.data import register_tensor_from_anndata, synthetic_iid
+from scvi.external import RNAStereoscope, SpatialStereoscope
+
+
+def test_stereoscope(save_path):
+    dataset = synthetic_iid()
+    sc_model = RNAStereoscope(dataset)
+    sc_model.train(
+        n_epochs=1, frequency=1, vae_task_kwargs={"lr": 0.01}, train_size=1.0
+    )
+    sc_params = sc_model.get_params()
+
+    dataset.obs["indices"] = np.arange(dataset.n_obs)
+    register_tensor_from_anndata(dataset, "ind_x", "obs", "indices")
```

Comment:
these two lines can be moved into the init of the highl-level model class. I would make the obs key name "_indices" instead so it doesn't likely interfere with anything.

Please be sure to add a test here for save + load. When you do that, use the `save_path` variable here as the root because it's a temporary directory.